src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -96,8 +97,23 @@ const getId = ({ pageType, service, variant, env }: GetIdProps) => {
       getIdFunction = () => pageType;
       break;
     case LIVE_PAGE:
+      getIdFunction = (path: string) => {
+        const isTipoId = isTipoIdCheck(path);
+        const isCpsId = isCpsIdCheck(path);
+        const id = isTipoId ? getTipoId(path) : getCpsId(path);
+        if (isTipoId) {
+          return `${id}`;
+        }
+        if (isCpsId) {
+          return `/${service}/live/${id}`;
+        }
+        return null;
+      };
+      break;
+
     case TOPIC_PAGE:
       getIdFunction = (path: string) => {
+        // Do we need some ID validation here?
```

Comment:
No, the ID validation already happens in Belfrage, so no need for it here
We know that we're rendering a live page because of the existence of `/live/` in the URL

Reply:
Removed